README.md

```diff
@@ -81,7 +81,7 @@ module.exports = {
     },
     {
       files: ['**/*.gts'],
-      parser: 'eslint-plugin-ember/gjs-gts-parser',
+      parser: 'ember-eslint-parser',
```

Comment:
Can we link to https://github.com/NullVoxPopuli/ember-eslint-parser in this section of the README?

Reply:
done